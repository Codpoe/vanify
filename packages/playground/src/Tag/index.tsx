import React from 'react';
import './index.less';

export const Tag: React.FC = () => {
  return <span className="tag">Hello Tag</span>;
};

export default Tag;
