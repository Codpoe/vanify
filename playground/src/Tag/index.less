.tag {
  display: inline-flex;
  align-items: center;
  border: 1px solid #ff0050;
  border-radius: 2px;
  height: 24px;
  font-size: 12px;
}
