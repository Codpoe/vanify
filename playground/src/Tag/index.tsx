import React from 'react';

export const Tag: React.FC = () => {
  return <span className="tag">Hello Tag</span>;
};

export default Tag;
