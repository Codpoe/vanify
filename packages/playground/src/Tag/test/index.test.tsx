import React from 'react';
import { render } from '@testing-library/react';
import Tag from '..';

test('render and match snapshot', () => {
  const result = render(<Tag />);
  expect(result.container).toMatchSnapshot();
});
