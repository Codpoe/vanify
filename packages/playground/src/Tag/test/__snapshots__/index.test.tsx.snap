// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and match snapshot 1`] = `
<div>
  <span
    class="tag"
  >
    Hello Tag
  </span>
</div>
`;
